README.md

```diff
@@ -4,49 +4,230 @@ Provides resources for installing and configuring both PowerDNS authoritative an
 
 ## Requirements
 
-IMPORTANT: Please read the Compatibility Notes version below since there is breaking changes between 2 and 3 versions of this cookbook.
+IMPORTANT: Please read the Deprecations and Compatibility Notes sections below since there is breaking changes between 2 and 3 versions of this cookbook.
```

Comment:
s/is/are/ and s/breaking changes between 2 and 3 versions/breaking changes between version 2 and 3/